apps/sms/views/conversation/js/startup.js

```diff
@@ -0,0 +1,87 @@
+/* global LazyLoader,
+          MessageManager,
+          Navigation,
+          TimeHeaders,
+          Settings,
+          LocalizationHelper,
+          ConversationView,
+          Information,
+          Threads,
+          Utils,
+          InterInstanceEventDispatcher
+*/
+
+(function(exports) {
+  'use strict';
+
+  const DEFAULT_PANEL = 'composer';
+
+  const LAZY_DEPENDENCIES = [
+    '/shared/js/settings_listener.js',
+    '/shared/js/mime_mapper.js',
+    '/shared/js/option_menu.js',
+    '/shared/js/mobile_operator.js',
+    '/shared/elements/gaia_sim_picker/script.js',
+    '/views/shared/js/waiting_screen.js',
+    '/views/shared/js/errors.js',
+    '/views/shared/js/dialog.js',
+    '/views/shared/js/error_dialog.js',
+    '/views/conversation/js/link_action_handler.js',
+    '/views/shared/js/contact_renderer.js',
+    '/views/shared/js/activity_picker.js',
+    '/views/conversation/js/information.js',
+    '/views/shared/js/localization_helper.js'
+  ];
+
+  function initLazyDependencies() {
+    return LazyLoader.load(LAZY_DEPENDENCIES).then(() => {
+      LocalizationHelper.init();
+      TimeHeaders.init();
+      Information.initDefaultViews();
+      Settings.init();
+
+      Navigation.setReady();
+
+      InterInstanceEventDispatcher.connect();
+    });
+  }
+
+  function initHeaders() {
+    var headers = document.querySelectorAll('gaia-header[no-font-fit]');
+    for (var header of headers) {
+      header.removeAttribute('no-font-fit');
+    }
+  }
+
+  function initShims() {
+    exports.InboxView = Object.freeze({
+      whenReady: () => Promise.resolve(),
+      markReadUnread: () => {},
+      updateThread: () => {},
+      isShim: true
```

Comment:
This is how I resolved this issue in my navigation WIP:

https://github.com/julienw/gaia-navigator/blob/sms-like-example-2/examples/sms-like/navigation.js#L23
https://github.com/julienw/gaia-navigator/blob/sms-like-example-2/examples/sms-like/navigation.js#L12 (I don't actually use this yet -- but I could use it in https://github.com/julienw/gaia-navigator/blob/sms-like-example-2/examples/sms-like/navigation.js#L59, not sure yet... maybe it's just duplicated data I should get rid of :) )